alert-policies/aws-storage-gateway/AvailabilityNotifications.yml

```
@@ -0,0 +1,28 @@
+name: Health notifications
+
+description: |+
+  This alert is triggered if the there are availability-related health notifications generated by the gateway
+
+type: STATIC
+nrql:
+  query: "SELECT count(aws.storagegateway.AvailabilityNotification) as 'Query' FROM Metric"
```

Its not that much important but please mention related count label like
```suggestion
  query: "SELECT count(aws.storagegateway.AvailabilityNotification) as 'AvailabilityNotificationCount' FROM Metric"
```

---

addresses